$NetBSD: patch-src_meson.build,v 1.1 2023/08/22 16:33:48 schmonz Exp $

Generate .gir using the built libraries, even if an earlier version is
already installed. Fixes update builds on macOS when e.g. new symbols
are introduced.

--- src/meson.build.orig
+++ src/meson.build
@@ -1280,6 +1280,9 @@
       export_packages: ['harfbuzz-gobject', 'harfbuzz'],
       header: 'hb-gobject.h',
       install: true,
+      env: {
+        'DYLD_LIBRARY_PATH': meson.project_build_root() + '/src'
+      },
       extra_args:  ['--cflags-begin',
                     '-DHB_NO_SINGLE_HEADER_ERROR',
                     '-DHAVE_GOBJECT',
